blockchain/storage/keyvalue.go

```diff
@@ -355,3 +355,7 @@ func (s *KeyValueStorage) Close() error {
 func (s *KeyValueStorage) NewBatch() Batch {
 	return s.db.NewBatch()
 }
+
+func (s *KeyValueStorage) WriteBatch() error {
+	return s.db.NewBatch().Write()
```

Comment:
I think that method isn't really necessary. I removed WriteBatch from the Storage interface because it is a helper method. I just forgot to remove it from the KeyValueStorage.